internal/uvm/create.go

```diff
@@ -122,6 +122,28 @@ type Options struct {
 	NumaProcessorCounts []uint32
 	// NumaMemoryBlocksCounts are the number of memory blocks per vNUMA node.
 	NumaMemoryBlocksCounts []uint64
+
+	EnableGraphicsConsole bool   // If true, enable a graphics console for the utility VM
+	ConsolePipe           string // The named pipe path to use for the serial console (COM1).  eg \\.\pipe\vmpipe
+}
+
+func verifyWCOWBootFiles(bootFiles *WCOWBootFiles) error {
```

Comment:
maybe make this a method and move it to `create_wcow.go`?
```suggestion
func (bf *WCOWBootFiles) verify() error {
  if bf == nil { return nil }
  // ...
```

Reply:
I prefer to keep this type as a plain struct than a type with methods, but don't have a strong preference. Let me know if you feel otherwise.